src/common/config/can_show_page.ts

```diff
@@ -0,0 +1,18 @@
+import { isComponentLoaded } from "./is_component_loaded";
+import { PageNavigation } from "../../layouts/hass-tabs-subpage";
+import { HomeAssistant } from "../../types";
+
+export const canShowPage = (hass: HomeAssistant, page: PageNavigation) => {
+  return (
+    (isCore(page) || isLoadedIntegration(hass, page)) &&
+    !hideAdvancedPage(hass, page)
+  );
+};
+
+const isLoadedIntegration = (hass: HomeAssistant, page: PageNavigation) =>
+  !page.component || isComponentLoaded(hass, page.component);
+const isCore = (page: PageNavigation) => page.core;
```

Comment:
You really like a function for EVERYTHING 🤣

Reply:
To be fair, this was an exceptional case, even for me :P Couldn't understand the boolean